app/utils/langchain-utils.js

```diff
@@ -130,6 +130,15 @@ const validateJSONResponse = (response) => {
   }
 }
 
+/**
+ * Validates and formats a JSON response.
+ *
+ * @param {Object} response - The response object to validate and format.
```

Comment:
I'm a bit confused with this jsdoc, if you want the actual response object from langchain's invoke function, it would look something like this

```
{
	"chat_history": []
	"context": [ { 'pageContent': '...' } ]
	"answer": "{ 'answer': '...', items: [] }"
}
```

Translated to jsdoc, you'd want
```
@param {{ chat_history: [], context: [], answer: string }} response
@returns {{ chat_history: [], context: [], answer: string }}
```